src/backend/config/configuration.template.yaml

```diff
@@ -37,6 +37,9 @@ tools:
   google_drive:
     client_id:
     client_secret:
+  google_web_search:
```

Comment:
The api_key should be in the secrets template file.

For the `cse_id` I'm not too sure - if it's not sensitive info then you can keep it in the config file